arangod/Agency/AsyncAgencyComm.cpp

```diff
@@ -432,6 +432,20 @@ arangodb::AsyncAgencyComm::FutureResult agencyAsyncSend(
 
 namespace arangodb {
 
+futures::Future<consensus::index_t> AsyncAgencyComm::getCurrentCommitIndex()
+    const {
+  auto future = sendWithFailover(fuerte::RestVerb::Get, "_api/agency/config",
```

Comment:
Nit: in most other places that call agency REST APIs, the paths start with a forward slash. Here the path doesn't start with a forward slash. It probably still works, but it would be more consistent to make the path start with a `/` here as well.
```suggestion
  auto future = sendWithFailover(fuerte::RestVerb::Get, "/_api/agency/config",
```